### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
+    return count
```
Comment (pavlejviki) [inline app/main.py:21]: try to solve in one line using count() method

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — CHANGES_REQUESTED** [review]
delete pycashe from git

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
rollback changes in .gitignore

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,5 @@
 .DS_Store
 venv/
 .pytest_cache/
+<<<<<<< HEAD
```
Comment (pavlejviki) [inline .gitignore:8]: remove redundant code

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — APPROVED** [review]
GJ!

**Pull request closed without merging**